src/server.ts

```diff
@@ -34,10 +37,17 @@ export class Server {
     this.initSockets()
     this.initTransporter()
     this.routes()
+    if (this.isProduction) {
```

Comment:
No, this is Patrick !

Reply:
J'ai mis ça pcq sinon j'ai un bug chelou ou j'peux pas import mes schemas pour le ws server